README.md

```diff
@@ -27,6 +27,7 @@
  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  -->
+<img src="doc/source/logo/NeuroM.jpg"/>
```

Comment:
I think the format is: `Format: ![Alt Text](url)`

Reply:
```suggestion
![NeuroM Logo](doc/source/logo/NeuroM.jpg)
```

Reply:
It seems to work anyway though:
https://github.com/jdcourcol/NeuroM/tree/banner

Reply:
I changed with Mike's suggestion. Both ways are working.